### PR TITLE
Open links in default browser

### PR DESCRIPTION
**Pull request by akrabat**
Implement /openurl to open links in a browser

Introduce the concept of local commands. The first one supported is `/openurl` which takes a list of link numbers which are displayed next to each link in the message view. Executing `/openurl` opens each link in the default browser.

Looks as shown here:
openlinks

### CODE REVIEW

**Comment by akrabat** [comment]
You'll probably have opinions on how this is implemented. I'm happy to modify as appropriate.

File: Sources/Application.swift [hunk 1]
```diff
@@ -283,5 +288,24 @@ class Application {
...
+    ///
+    /// - /openurl {space separated list of numbers}: opens numbered link in default browser
+    ///
+    func executeLocalCommand(_ command: String) -> Bool {
+        if (command.hasPrefix("/openurl")) {
+            let pieces = command.components(separatedBy: [",", " "])
+            let linkNumbers = pieces.flatMap { Int($0) }
+            for linkNumber in linkNumbers {
+                let url = self.links[linkNumber - 1]
+                NSWorkspace.shared().open(NSURL(string: url)! as URL)
```
Comment (glock45) [inline Sources/Application.swift]: Let's switch to Util.shell("open", "url"). I think AppKit/NSWorkspace.shared().open will not work on Linux.

**Comment by glock45** [comment]
I've added a comment: https://github.com/slash-hq/slash/pull/4/files#r93789655

File: Sources/Application.swift (hunk 1 above)
Comment (akrabat) [inline Sources/Application.swift]: Done. I think that the command will be different on Linux though, so at some point that'll have to be a configuration option.

**Pull request merged**